### PR TITLE
Model read me

### PR DESCRIPTION
**Pull request by thatmikeparrish**
## Link to Ticket
Closes # [Ticket](https://github.com/NSS-Therapeutic-RaccoonsBangazonAPI/issues/14)

## Description of Proposed Changes
- Created Models for Computer, Customer, Department, Employee, Order.
- Created a README.md

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <ModelReadMe>
```

## Mentions 

@thatmikeparrish 
@ricky-bruner 
@jeremiah3643 
@KHardt 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.

### CODE REVIEW

**Review by jeremiah3643 — DISMISSED** [review]
👍  Didn't break anything!

**Review by askingalot — CHANGES_REQUESTED** [review]
Why is the EmployeeController in the "models" PR?

You have a mix of `BangazonAPI.Data` and `BangazonAPI.Models` for your model namespaces.

The readme looks good

File: BangazonAPI/Controllers/EmployeesController.cs [hunk 1]
```diff
@@ -0,0 +1,180 @@
+﻿/*
```
Comment (askingalot) [inline BangazonAPI/Controllers/EmployeesController.cs]: Why is the EmployeeController in the "models" PR?

**Review by askingalot — CHANGES_REQUESTED** [review]

File: BangazonAPI/Controllers/EmployeesController.cs (hunk 1 above)
Comment (thatmikeparrish) [reply BangazonAPI/Controllers/EmployeesController.cs]: I must have made a branch off of a branch. I'll fix that.

Comment (thatmikeparrish) [reply BangazonAPI/Controllers/EmployeesController.cs]: Ok, I put the employee controller in the correct branch and removed it from this one.

**Review by askingalot — APPROVED** [review]
This is a good foundation. Remember, though, that you *might* have to make changes to these models as you go along. Not sure that you will, but you may have to....Just something to keep in mind as you work on future tickets.

**Pull request merged**